framework/ui/src/main/java/com/google/android/catalog/framework/ui/components/AdaptivePane.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.catalog.framework.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun AdaptivePane(
+    paddingValues: PaddingValues,
+    start: @Composable () -> Unit,
+    end: @Composable () -> Unit
+) {
+    if (isExpandedScreen()) {
+        Row(Modifier.fillMaxSize()) {
```

Comment:
You could consider using accompanist/TwoPane here!

Reply:
Oh I was not aware of it. I might look into it, but for now I prefer not to add experimental libraries since it increases the maintainance and might conflict with other dependencies. This is an SDK so it will be used by others